model/clusters_mgmt/v1/azure_node_pool_os_disk_type.model

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// AzureNodePoolOsDisk defines the os disk attributes settings.
+struct AzureNodePoolOsDisk {
+	// The size in GiB to assign to the OS disks of the
+    // Nodes in the Node Pool. The property
+    // is the number of bytes x 1024^3.
+    // If not specified, OS disk size is 30 GiB.
+    SizeGibibytes Integer
+
+    // The disk storage account type to use for the OS disks of the Nodes in the
+    // Node Pool. Valid values are:
+    // * Standard_LRS: HDD
+    // * StandardSSD_LRS: Standard SSD
+    // * Premium_LRS: Premium SDD
+    // * UltraSSD_LRS: Ultra SDD
+    //
+    // If not specified, `Premium_LRS` is used.
+    StorageAccountType String
+
+    // Specifies the type of OS disk for the nodes in the Node Pool.
+    // Valid values ared:
+    // * Persistant
+    // * Ephemeral
+    // If not specified, Persistant OS Disks are used.
+    Persistance String
```

Comment:
The correct spelling is `Persistence`

Same for the accepted value. Persistant -> Persistent